apps/music/views/playlists/view.js

```diff
@@ -32,8 +32,8 @@ var PlaylistsView = View.extend(function PlaylistsView() {
 
   this.searchResults.getItemImageSrc = (item) => this.getThumbnail(item.name);
 
-  this.list.scrollTop = this.searchBox.HEIGHT;
   this.list.minScrollHeight = `calc(100% + ${this.searchBox.HEIGHT}px)`;
+  this.list.offset = this.searchBox.HEIGHT;
```

Comment:
Yeah it's being done in the `view.js` `.setupList()` abstraction. We can't use that in this view as there are some subtle differences.